unity-renderer/Assets/Scripts/MainScripts/DCL/AvatarSystem/AvatarSystemUtils.cs

```diff
@@ -68,9 +68,15 @@ public static void PrepareMaterialColors(Rendereable rendereable, Color skinColo
             {
                 foreach (Material material in renderer.materials)
                 {
-                    if (material.name.ToLower().Contains("skin"))
+                    // If this is modified, check DecentralandMaterialGenerator.SetMaterialName,
+                    // its important for the asset bundles materials to have normalized names but this functionality should work too
+
+
+                    string name = material.name.ToLower();
+
+                    if (name.Contains("skin"))
```

Comment:
you can use `StringComparison.OrdinalIgnoreCase` instead of creating a new instance of the `string`